docker-compose.yml

```diff
@@ -39,13 +39,17 @@ services:
   transaction-service:
       build:
           context: ./backend/transaction
-      container_name: transaction
+      container_name: transaction-service
+      volumes:
+        - ./backend/middleware:/backend/middleware
       ports:
         - "5000:5000"
```

Comment:
can u change to 3004:3004 so that it's similar to other services

Reply:
ok